pkgs/applications/science/logic/cadical/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "cadical";
+  version = "1.2.1";
+
+  src = fetchFromGitHub {
+    owner = "arminbiere";
+    repo = pname;
+    rev = "92d72896c49b30ad2d50c8e1061ca0681cd23e60";
```

Comment:
This should point to a release tag instead of a commit hash. Otherwise automatic updates won't work and its easy to get a mismatch between `version` and `rev`.

Reply:
```suggestion
    rev = "rel-${version}";
```

Reply:
Thanks for the review and explanation :)

Reply:
Thank you for reviewing! I revised it.